keyboards/dp60/keymaps/indicator/indicator.c

```diff
@@ -0,0 +1,224 @@
+/*
```

Comment:
I'm not sure that this file is needed, especially with the newer "lighting layers" code.

https://docs.qmk.fm/#/feature_rgblight?id=lighting-layers

Reply:
Yes, seems it can be used with the "lighting layers" codes, but it requires to customize the rgblight_set() function because a different pin was used for the leds on the plate.

Reply:
Currently the RGBLIGHT_LAYERS can not use with RGBLIGHT_CUSTOM_DRIVER due to a link error.
 ```
Compiling: quantum/rgblight.c                                                                      quantum/rgblight.c:638:13: error: 'rgblight_layers_write' defined but not used [-Werror=unused-function]
 static void rgblight_layers_write(void) {
             ^
cc1.exe: all warnings being treated as errors
```
It was caused by the current implementation of the light layers.
```
#ifndef RGBLIGHT_CUSTOM_DRIVER
void rgblight_set(void) {
    LED_TYPE *start_led;
    uint16_t  num_leds = clipping_num_leds;

#    ifdef RGBLIGHT_LAYERS
    if (rgblight_layers != NULL) {
        rgblight_layers_write();
    }
#    endif
```
riglight_layers_write() was a static function and can't be called out of rgblight.c

Reply:
Ah. 

No need to use custom driver anymore.  you can use `rgblight_call_driver` and handle it there. 

You can take a look at how the ergodox ez does this in the led_i2c.c file, in fact.